riscv-rt/src/interrupts.rs

```diff
@@ -81,7 +81,7 @@ core::arch::global_asm!(
         .type _vector_table, @function
         
         .option push
-        .balign 0x4 // TODO check if this is the correct alignment
+        .balign 0x100 // RISC-V requires 4, but implementations may be stricter
```

Comment:
I really do not think we should add such strict alignment requirements based on a single 32-bit RISC-V core implementation.

Reply:
Added cargo features for control.